### PR TITLE
Unexplored regions in Ship Modal

### PR DESCRIPTION
**Pull request by AevitasDragonkin**
## Description

Instead of only listing the regions you have unlocked. It will display all regions that are currently available, and display the ones you have not unlocked yet as "Unexplored region".
There was the option to add "More regions coming soon", but the general gist in discord was that this would probably raise more questions of when the new content would be coming. So we're leaving it out for now.

## Motivation and Context

Lots of bug reports recently of people completing Galar and asking why they can't get into Hisui. This should hopefully make it clear that there currently is no region implemented after Galar. As soon as Hisui is released, this will automatically add Hisui as "Unexplored region".

## How Has This Been Tested?

* Opened the ship modal on a save file with only Hoenn unlocked, showing the Unexplored regions.
* Openend it with a completed save file, showing all regions fully unlocked. 
* Openend it with Hisui enabled on a completed save file and it showed correctly as Unexplored region, and switching to Hisui correctly after first traveling to Hisui.

## Screenshots (optional):

Went with the second option
image

## Types of changes

- UI improvement

### CODE REVIEW

File: src/components/shipModal.html [hunk 1]
```diff
@@ -9,29 +9,47 @@ <h5 class="modal-title">Regional Travel</h5>
...
+                            <!-- /ko -->
                         <!-- /ko -->
-                    </div>
                     <!-- /ko -->
+<!--                    <div class="btn-group w-100 my-1">-->
+<!--                        <button style="position: relative;" class='btn btn-block btn-secondary ship-modal-region-button'-->
+<!--                                data-bind='attr: { disabled: true }'>-->
+<!--                            <knockout>More regions coming soon™</knockout>-->
+<!--                        </button>-->
+<!--                    </div>-->
```
Comment (RegisCoaxans) [inline src/components/shipModal.html:52]: Can you remove that, please ?

**Review by RegisCoaxans — APPROVED** [review]
I tested it and looked to the code. Seems good to me.

File: src/components/shipModal.html (hunk 1 above)
Comment (AevitasDragonkin) [reply src/components/shipModal.html]: Done

**Comment by RedSparr0w** [comment]
What option did you end up going with in this PR?
Don't remember what was discussed on the Discord 😅

**Review by RedSparr0w — APPROVED** [review]
nvm, just re-read the description.

Looks good, thanks for the PR! 👍

**Comment by AevitasDragonkin** [comment]
Went with option 2
image

**Pull request merged**